files/ko/web/javascript/reference/global_objects/string/trimstart/index.html

```diff
@@ -0,0 +1,106 @@
+---
+title: String.prototype.trimStart()
+slug: Web/JavaScript/Reference/Global_Objects/String/trimStart
+tags:
+- JavaScript
+- Method
+- Prototype
+- Reference
+- String
+- Polyfill
+browser-compat: javascript.builtins.String.trimStart
+translation_of: Web/JavaScript/Reference/Global_Objects/String/trimStart
+---
+<div>{{JSRef}}</div>
+
+<p><strong><code>trimStart()</code></strong>메서드는 문자열 시작부분의 공백을 제거합니다. 또한 <code>trimLeft()</code>라는 별칭으로 호출이 가능합니다.</p>
+
+<div>{{EmbedInteractiveExample("pages/js/string-trimstart.html")}}</div>
+
+
+<h2 id="syntax">구문</h2>
+
+<pre class="brush: js">
+trimStart()
+
+trimLeft()
+</pre>
+
+<h3 id="return_value">반환값</h3>
+
+<p><code><var>str</var></code>시작부분(왼쪽)에서 공백이 제거된 새 문자열을 반환합니다.</p>
+
+<p>If the beginning of <code><var>str</var></code>에 공백이 없을시에도 에러가 발생하지 않고 여전히 새 문자열(본질적으로 <code><var>str</var></code>의 복사본)이 반환됩니다.</p>
```

Comment:
```suggestion
<p><code><var>str</var></code>에 공백이 없을시에도 에러가 발생하지 않고 여전히 새 문자열(본질적으로 <code><var>str</var></code>의 복사본)이 반환됩니다.</p>
```

'If the beginning of '은 빠져야 할거 같아요!